lib/board.rb

```diff
@@ -1,42 +1,64 @@
 class Board
 
-  def create_grid(dimension)
-    size = Math.sqrt(dimension)
+  attr_reader :dimension, :grid
```

Comment:
If these are used publicly then it is fine to have them here, but if not, I'd put them in the private section.